app/models/version.rb

```diff
@@ -133,6 +133,11 @@ def yanked?
     !indexed
   end
 
+  def deleted?
+    key = "gems/#{full_name}.gem"
```

Comment:
Can we extract this into a method? It's used all over `Version` and in `Deletion` too iirc